fog/view/enclave/impl/src/lib.rs

```diff
@@ -75,6 +75,34 @@ impl Default for BlockData {
     }
 }
 
+/// Helper struct that contains data associated with the "last known" fields in
+/// the `QueryResponse`.
+struct LastKnownData {
```

Comment:
`impl Default for u64` will be zero, so this can be derived.

```suggestion
#[derive(Default)]
struct LastKnownData {
```

Reply:
Done.